dev-python/pytest-describe/pytest-describe-0.11.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
```

Comment:
What about 3.7, I guess you're on a stable system?

Reply:
I missed python3_7 for this package, I'll add it.